services/web/pkg/config/options.go

```diff
@@ -27,6 +27,7 @@ type Options struct {
 	TokenStorageLocal        bool             `json:"tokenStorageLocal" yaml:"tokenStorageLocal" env:"WEB_OPTION_TOKEN_STORAGE_LOCAL" desc:"Specifies whether the access token will be stored in the local storage when set to 'true' or in the session storage when set to 'false'. If stored in the local storage, login state will be persisted across multiple browser tabs, means no additional logins are required."`
 	DisabledExtensions       []string         `json:"disabledExtensions,omitempty" yaml:"disabledExtensions" env:"WEB_OPTION_DISABLED_EXTENSIONS" desc:"A list to disable specific Web extensions identified by their ID. The ID can e.g. be taken from the 'index.ts' file of the web extension. Example: 'com.github.owncloud.web.files.search,com.github.owncloud.web.files.print'. See the Environment Variable Types description for more details."`
 	Embed                    *Embed           `json:"embed,omitempty" yaml:"embed"`
+	UserListRequiresFilter   bool             `json:"userListRequiresFilter,omitempty" yaml:"userListRequiresFilter" env:"WEB_OPTION_USER_LIST_REQUIRES_FILTER" desc:"Defines whether one ore more filters must be set in order to list users in the Web admin settings. Set this option to 'true' if running in an environment with a lot of users and listing all users would be slow performance wise. Defaults to 'false'."`
```

Comment:
```suggestion
	UserListRequiresFilter   bool             `json:"userListRequiresFilter,omitempty" yaml:"userListRequiresFilter" env:"WEB_OPTION_USER_LIST_REQUIRES_FILTER" desc:"Defines whether one ore more filters must be set in order to list users in the Web admin settings. Set this option to 'true' if running in an environment with a lot of users and listing all users could slow down performance. Defaults to 'false'."`
```

Reply:
Urgh, I think I made a mistake here, committing this. 
The sentence states now that listing all users via the admin ui would slow down the whole system. I don't think we have proof for that.

Reply:
Make a new PR and add me plus reference this PR so we can see the follow up